src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
```suggestion
    void draw();
```
this method should call System.out.println()